File: JavaScript/Faccat/Exercicio7.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite a quantidade de anos: ', function(anos) {
  rl.question('Digite a quantidade de meses: ', function(meses) {
    rl.question('Digite a quantidade de dias: ', function(dias) {
      const conversao = anos * 365 + meses * 30 + dias;
      console.log('A conversão da idade em dias é: ' + conversao);
      
      rl.close();
    });
  });
});

File: Java/src/Faccat/Exercicio7.java
package Faccat;

import java.util.Scanner;

public class Exercicio7 {
    public static void main(String[] args) {
        int anos, meses, dias, conversao;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite a quantidade de anos: ");
        anos = scanner.nextInt();

        System.out.print("Digite a quantidade de meses: ");
        meses = scanner.nextInt();

        System.out.print("Digite a quantidade de dias: ");
        dias = scanner.nextInt();

        conversao = anos * 365 + meses * 30 + dias;

        System.out.println("A conversão da idade em dias é: " + conversao);

        scanner.close();
    }
}
